src/utils/naming.ts

```diff
@@ -15,9 +15,7 @@ export const setCloneName = (name: string, date?: Date): string => {
   const hour = d.getUTCHours()
   const minutes = d.getUTCMinutes()
   const seconds = d.getUTCSeconds()
-  return `${name.trim()} (cloned at ${prependZero(month)}-${prependZero(
+  return `${name.trim()} (cloned at ${year}-${prependZero(month)}-${prependZero(
     day
-  )}-${year}:${prependZero(hour)}:${prependZero(minutes)}:${prependZero(
-    seconds
-  )})`
+  )} ${prependZero(hour)}:${prependZero(minutes)}:${prependZero(seconds)})`
```

Comment:
`YYYY-MM-DD HH:MM:SS` - this allows both Chrome and Firefox to parse this string and create a `Date` with it

Reply:
Let’s hold off until @garylfowler can give some input on this since the original call was made with that format in mind